src/views/HomeView.vue

```diff
@@ -1,41 +1,66 @@
 <template>
   <div class="home-container">
-    <h1>Schule 4.0 - Haustiere</h1>
-    <p>Wähle ein Haustier um das Spiel zu starten:</p>
-    <select v-model="selectedPet" @change="goToPetStages">
-      <option disabled value="">Wähle ein Haustier</option>
-      <option value="dog">Hund</option>
-    </select>
+    <div class="header-box">
+      <h1>Mein erstes Haustier</h1>
+      <img src="../assets/icon_audio.svg" alt="Audio">
+    </div>
+
+    <p>Bereit, in die Welt der Haustiere einzutauchen?
+    <br>Tippe auf das Tier, das dich am meisten interessiert!</p>
+  </div>
+
+  <div class="pet-cards">
+      <PetCardComponent v-for="(pet, index) in pets" :key="index" :pet="pet" @pet-selected="handlePetSelected" />
```

Comment:
Handle the logic what happens when clicking on the PetCardComponent directly in the component itself instead of passing the handlePetSelected function.
(Note: You don't need to add "Component" behind custom components "PetCard.vue" would be enough)